packages/admin/admin/src/index.ts

```diff
@@ -8,6 +8,7 @@ export { AppHeaderMenuButton, AppHeaderMenuButtonClassKey, AppHeaderMenuButtonPr
 export { buildCreateRestMutation, buildDeleteRestMutation, buildUpdateRestMutation } from "./buildRestMutation";
 export { readClipboardText } from "./clipboard/readClipboardText";
 export { writeClipboardText } from "./clipboard/writeClipboardText";
+export { Accordion } from "./common/Accordion";
```

Comment:
When using a component library in typescript, you generally expect to have access to its component types.  
Please also export `AccordionComponentsProps`, `AccordionProps` and (after it's added) `AccordionClassKey`.